.github/workflows/azure-devops-pipelines-cg.yml

```diff
@@ -0,0 +1,49 @@
+# This pipeline is intended to use inside Azure DevOps for Component Governance,
+# which is may not useful for users outside Microsoft.
+
+variables:
+- name: BuildParameters.solution
+  value: '**\*.sln'
+  
+# This pipeline will be triggered when either main branch is pushed or 2AM on workdays.
+trigger:
+  branches:
+    include:
+    - main
+  batch: True
+schedules:
+- cron: 0 2 * * 1,2,3,4,5
+  branches:
+    include:
+    - main
+ 
+name: $(date:yyyyMMdd)$(rev:.r)
+resources:
```

Comment:
Is this necessary here?

Reply:
The entire `resource` block I mean.

Reply:
Removed. It was auto generated by classic pipeline